clib/src/main/resources/scala-native/complex.c

```diff
@@ -1,3 +1,4 @@
+#if defined(__SCANATIVE_C_COMPLEX)
```

Comment:
```suggestion
#if defined(__SCALANATIVE_C_COMPLEX)
```

Reply:
Looks like a duplicate?

Reply:
Thank you, I don't think it's duplicate, there is one type in preprocessor guards and one in @define annotation